### ISSUE TITLE
Play notification sound only once for rapidly arriving messages (especially in group chats)

### ISSUE THREAD

**Issue by cgogolin** [WhisperSystems/Signal-Android#5410]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
The notification sound is played many times in rapid succession when many messages have piled up (particularly in group chats) and the device receives them all "at once" after beeing offline. This is unnecessary and annoying. 

### Steps to reproduce
- go offline with some device
- send several messages to the offline device
- go online
- *pling* *pling* *pling*... 

**Actual result:** 
Only one *pling* per conversation with new messages.

**Comment by FeuRenard** [comment]
This issue is already tracked by three VERY similar issues: #778, #3165 & #5101.

**Comment by 2-4601** [comment]
dupe

**Comment by cgogolin** [comment]
I am sorry for not finding the duplicates.
